From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Juho=20H=C3=A4m=C3=A4l=C3=A4inen?=
 <juho.hamalainen@seafarix.com>
Date: Mon, 7 Oct 2024 10:49:22 +0300
Subject: [PATCH] appsupport: Add
 application/vnd.android.package-archive-bundle
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Juho Hämäläinen <juho.hamalainen@seafarix.com>
---
 data/freedesktop.org.xml.in | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/data/freedesktop.org.xml.in b/data/freedesktop.org.xml.in
index 1dc1530..b55f9fa 100644
--- a/data/freedesktop.org.xml.in
+++ b/data/freedesktop.org.xml.in
@@ -1376,6 +1376,13 @@ command to generate the output files.
     <sub-class-of type="application/java-archive"/>
     <glob pattern="*.apk"/>
   </mime-type>
+  <mime-type type="application/vnd.android.package-archive-bundle">
+    <comment>Android package bundle</comment>
+    <generic-icon name="package-x-generic"/>
+    <sub-class-of type="application/zip"/>
+    <glob pattern="*.apkm"/>
+    <glob pattern="*.xapk"/>
+  </mime-type>
   <mime-type type="application/vnd.symbian.install">
     <comment>SIS package</comment>
     <acronym>SIS</acronym>
